CHANGELOG.md

```diff
@@ -50,6 +50,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The builtin `jj update` and `jj up` aliases for `jj checkout` have been
   deleted.
 
+* Change IDs are now in rendered using letters from the end of the alphabet
+  (backwards from 'z' through 'k') instead of the usual hex digits ('0' through
```

Comment:
I'm not sure the "backwards" is useful, I think `(from 'k' to 'z')` would be enough and a bit less confusing.